contracts/contracts/LineaRollup.sol

```diff
@@ -150,6 +150,20 @@ contract LineaRollup is
     emit LineaRollupVersionChanged(bytes8("5.0"), bytes8("6.0"));
   }
 
+  /**
+   * @notice Revokes `role` from the calling account.
+   * @dev Fallback operator cannot renounce role. Reverts with OnlyNonFallbackOperator.
+   * @param _role The role to renounce.
+   * @param _account The account to renounce - can only be the _msgSender().
+   */
+  function renounceRole(bytes32 _role, address _account) public virtual override {
```

Comment:
Do we want to put virtual here ?

Reply:
removed - was keeping the original signature for clarity, but we don't really need it.

Reply:
perfect thanks :)